swagger.yaml

```diff
@@ -3404,6 +3404,16 @@ paths:
           type: string
           format: uuid
           required: false
+        - in: query
```

Comment:
We can add validation in swagger, reducing boiler plate code https://swagger.io/docs/specification/data-models/data-types/#range